src/node/http.ts

```diff
@@ -33,11 +34,11 @@ declare global {
 }
 
 export const createClientConfiguration = (req: express.Request): ClientConfiguration => {
-  const base = relativeRoot(req)
+  const base = relativeRoot(req.originalUrl)
 
   return {
-    base,
-    csStaticBase: normalize(path.posix.join(base, "_static/")),
+    base: base,
```

Comment:
nit: since these are the same name, we should be able to keep it `base` instead of `base: base`